lib/client.js

```diff
@@ -40,18 +40,19 @@ dav.Client.prototype = {
      *
      * @param {string} url Url to do the propfind request on
      * @param {Array} properties List of properties to retrieve.
+     * @param {Object} headers headers
```

Comment:
per jsdoc this should be `@param {Object} [headers] headers` as the newly added param is optional

http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values